default/stringtables/en.txt

```diff
@@ -1941,6 +1941,9 @@ Previous Player
 MULTIPLAYER_PLAYER_LIST_STARTING_SPECIES
 Starting Species
 
+EDITABLE_GALAXY_SETTINGS
+Anyone can edit galaxy and AI settings.
```

Comment:
This name is a bit long for the checkbox itself. Maybe "All Can Edit Settings" for the label, and then add an (even more so than this) longer tooltip explaining what the option does.